test-clients/src/test/java/E2EPackageRunner.java

```diff
@@ -294,12 +294,12 @@ Collection<DynamicContainer> contractPrecompile() {
 	@TestFactory
 	Collection<DynamicContainer> contractPrecompileEth() {
 		return List.of(new DynamicContainer[] {
-//				extractSpecsFromSuiteForEth(AssociatePrecompileSuite::new),
-//				extractSpecsFromSuiteForEth(ContractBurnHTSSuite::new),
-//				extractSpecsFromSuiteForEth(ContractHTSSuite::new),
-//				extractSpecsFromSuiteForEth(ContractKeysHTSSuite::new),
-//				extractSpecsFromSuiteForEth(ContractMintHTSSuite::new),
-				extractSpecsFromSuiteForEth(CreatePrecompileSuite::new)
+				extractSpecsFromSuiteForEth(AssociatePrecompileSuite::new),
+				extractSpecsFromSuiteForEth(ContractBurnHTSSuite::new),
+				extractSpecsFromSuiteForEth(ContractHTSSuite::new),
+				extractSpecsFromSuiteForEth(ContractKeysHTSSuite::new),
+				extractSpecsFromSuiteForEth(ContractMintHTSSuite::new),
+//				extractSpecsFromSuiteForEth(CreatePrecompileSuite::new)
```

Comment:
Not sure of team convention but if you leave comments we should add reason why it's commented out.
Hopefully this doesn't make it into master on merge

Reply:
No-op if there's no next commit though. Let's get this merged in. 
Nice work